api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -606,3 +616,23 @@ join \`$OUTPUT_PROJECT.$OUTPUT_DATASET.domain\` d2
 on d2.domain_id = c.domain_id
 and (c.count_value > 0 or c.source_count_value > 0)
 group by d2.domain_id,c.vocabulary_id"
+
+###############################################################
+# Update the path of concepts in achilles_results to fetch tree
```

Comment:
Could you expand this? I don't follow what fetch tree means.

Reply:
I was fetching the path of concepts Eg: essential hypertension which can have path clinical finiding -> Finding by site -> Disorder of cardiovascular system -> Hypertensive disorder -> Essential hypertension. Storing the concept id path of tree in here.

Reply:
Please add this comment to the code